### PR TITLE
Bugfix/86bz4fzma

### PR DESCRIPTION
**Pull request by NatalySheludko**
Create global styles with Material UI theme, remove styling approach styled-components

### CODE REVIEW

File: ui/src/styles/theme.ts [hunk 1]
```diff
@@ -1,99 +1,105 @@
...
-  greyGreenLightest: "#BDC7BF",
-  greyGreenPale: "#D8DEDA",
-  greyGreenVeryPale: "#F4F6F4",
+const theme = createTheme({
+  main: {
+    custom: {
+      systemInfo: '#363DFA',
+      systemWarning: '#FFD900',
+      systemError: '#F21111',
+			systemSuccess: '#11BF18',
```
Comment (y-andriievskyi) [inline ui/src/styles/theme.ts:19]: fix indentation

File: ui/src/styles/theme.types.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
+export interface CustomStyles {
+  custom: {
+    systemInfo: string;
+    systemWarning: string;
+    systemError: string;
+		systemSuccess: string;
```
Comment (y-andriievskyi) [inline ui/src/styles/theme.types.ts:6]: indentation

**Review by y-andriievskyi — CHANGES_REQUESTED** [review]

File: ui/src/styles/theme.ts (hunk 1 above)
Comment (NatalySheludko) [reply ui/src/styles/theme.ts]: fixed

File: ui/src/styles/theme.types.ts (hunk 2 above)
Comment (NatalySheludko) [reply ui/src/styles/theme.types.ts]: fixed

**Review by y-andriievskyi — APPROVED** [review]

**Review by anastasia1383 — APPROVED** [review]
Nice

**Pull request merged**